src/components/Layout/useLayoutState.tsx

```diff
@@ -56,6 +57,12 @@ type LayoutState = {
    * @param fullscreen - true to go fullscreen mode; false to go to non-fullscreen mode
    */
   setFullscreenMode: (fullscreen: boolean) => void;
+  /**
+   * Holds the Point where a drag-select action has been started.
+   * undefined if there is no drag-select action.
+   */
+  dragSelectStartCoords?: Point;
+  setDragSelectStartCoords: (point?: Point) => void;
```

Comment:
I understand that most of this `LayoutState` type has no code comments for other properties and setters. The change you introduced for the `fullScreenMode` feature added comments for the property and setter individually and now this change bundles both in the same comment.

Looking at this piece of code, I am seeing 3 distinct approaches and styles to comments, so I think we could settle on a single and consistent approach (even if it is not having any comments).

Reply:
I would say where it is valuable. In my opinion comments like „Set the drag select start coords“ don't make sense…

Anyway will see if I can improve here.

Reply:
Added a comment for `setDragSelectStartCoords` as well. 

In my opinion the rule should be „where it makes sense“. Otherwise you will end up with comments like „Set drag select coords“ :laughing: